cmd/lncli/commands.go

```diff
@@ -1400,6 +1402,38 @@ func channelBalance(ctx *cli.Context) error {
 	return nil
 }
 
+var generateManPageCommand = cli.Command{
+	Name:   "generatemanpage",
+	Usage:  "Generates a man page for lncli and lnd as " +
+		"lncli.1 and lnd.1 respectively.",
+	Hidden: true,
+	Action: actionDecorator(generateManPage),
+}
+
+func generateManPage(ctx *cli.Context) error {
+	// Generate the man pages for lncli as lncli.1.
+	manpages, err := ctx.App.ToMan()
+        if err != nil {
+		return err
+        }
+	err = os.WriteFile("lncli.1", []byte(manpages), 0644)
+	if err != nil {
+		return err
+	}
+
+	// Generate the man pages for lnd as lnd.1.
+	config := lnd.DefaultConfig()
+	fileParser := flags.NewParser(&config, flags.Default)
+	var buf bytes.Buffer
+	fileParser.WriteManPage(&buf)
+	err = os.WriteFile("lnd.1", buf.Bytes(), 0644)
```

Comment:
formatting nit: can we space things out here a bit pls?

```
        config := lnd.DefaultConfig()
	fileParser := flags.NewParser(&config, flags.Default)
	fileParser.Name = "lnd"
	
	var buf bytes.Buffer
	fileParser.WriteManPage(&buf)
	
	err = os.WriteFile("lnd.1", buf.Bytes(), 0644)
	if err != nil {
		return err
	}

	return nil
```

Reply:
Thanks! Addressed